.npmrc

```diff
@@ -1 +1,2 @@
 package-lock=false
+node-options=--max_old_space_size=2048
```

Comment:
do we still need this?

Reply:
I assume so? I remove for these scripts: https://github.com/cerner/terra-core/pull/3021/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L81-L83